pkg/collector/corechecks/gpu/nvidia/collector.go

```diff
@@ -52,11 +53,12 @@ const (
 
 // Metric represents a single metric collected from the NVML library.
 type Metric struct {
-	Name     string  // Name holds the name of the metric.
-	Value    float64 // Value holds the value of the metric.
-	Type     metrics.MetricType
-	Priority MetricPriority // Priority is the priority of the metric, indicating which metric to keep in case of duplicates. Low (default) is the lowest priority.
-	Tags     []string       // Tags holds optional metric-specific tags (e.g., process ID).
+	Name                string  // Name holds the name of the metric.
+	Value               float64 // Value holds the value of the metric.
+	Type                metrics.MetricType
+	Priority            MetricPriority   // Priority is the priority of the metric, indicating which metric to keep in case of duplicates. Low (default) is the lowest priority.
+	Tags                []string         // Tags holds optional metric-specific tags (e.g., process ID).
```

Comment:
perhaps worth to rephrase the comment, following the addition of the AssociatedWorkloads field (especially because in both examples process is used, which is a bit confusing)